### ISSUE TITLE
`declaration-colon-newline-after` takes the opening of a `/*/` comment for its end and writes the line break into the comment's text

### ISSUE THREAD

**Issue by firefoxic** [stylelint-stylistic/stylelint-stylistic#400]
### What minimal example or steps are needed to reproduce the bug?

`declaration-colon-newline-after` lets a comment stand on the colon's own line and asks for the break behind that comment rather than behind the colon. It finds the end of the comment by searching for `*/` from the colon:

```js
let indexToCheck = OPENS_WITH_BLOCK_COMMENT.test(propPlusColon.slice(i + 1)) ? propPlusColon.indexOf(`*/`, i) + 1 : i
```

A comment written as `/*/` opens on a solidus, and that search takes the solidus and the star it stands behind for the comment's end — three characters into a comment that has not closed yet. The break is then written inside the comment's text:

```css
a { b: /*/ x */y; }
```

comes back from `--fix` as

```css
a { b: /*/
 x */y; }
```

The warning is reported at `1:10`, inside the comment, rather than at `1:15` where the comment ends. Every syntax reads it the same way, and every head of that shape does: `/*/*a*/`, `/*/*/`, `/*/ */` and `/*//*/` all close early.

Nothing is destroyed — the output parses, the comment is still a comment, and a second run has nothing to add — but the file the user gets has a line break written into prose they wrote, and the code the rule was asked about is not where it was reported.

### What minimal configuration is needed to reproduce the bug?

```json
{
	"plugins": [
		"@stylistic/stylelint-plugin"
	],
	"rules": {
		"@stylistic/declaration-colon-newline-after": "always"
	}
}
```

`"always-multi-line"` reaches the same shapes.

### What did you expect to happen?

A comment closes at the first `*/` behind its own opening, so the break goes behind the whole of it and the warning stands where the comment ends.

### What actually happened?

The break is written three characters into the comment, and the warning is reported there.

### Notes

None of the five oracles can see this. `comments.mjs` counts openings, and the star written in front of the break gives the text behind it an opening of its own, so the tally is kept — the same blindness [#272](https://github.com/stylelint-stylistic/stylelint-stylistic/issues/272) ran into. `converge.mjs` sees a file that settles on the first run, `nodes.mjs` a tree with every node standing, and neither `control.mjs` nor `twins.mjs` turns on how a comment is spelled inside itself.

Found while closing [#371](https://github.com/stylelint-stylistic/stylelint-stylistic/issues/371), whose branch reaches the same defect on a second class of input — a value holding no word of its own, where the text this rule reads used to end at the colon and now runs to the end of the value — and answers it there rather than leaving the branch to widen it.
